### PR TITLE
Update storage amount

### LINKED ISSUE #1482: Update storage estimates
Storage estimates need to be updated. Link: https://besu.hyperledger.org/public-networks/concepts/data-storage-formats#syncing-nodes
@ahamlat might have recent numbers. Otherwise anyone in the engineering team can help you to get this numbers.

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
Closes #1482 

Updating doc to reflect new numbers detailed in [this doc](https://docs.google.com/document/d/1ykIANeVOh9E95-l-ZQU1L6QkyXwpaBcXhPEr7w3xuSs/edit#heading=h.vcpe6a3o9ru9). 

Should Checkpoint values also be added to this table?

### CODE REVIEW

**Comment by gfukushima** [comment]
The updated number matches with the numbers the doc has. But they are from 3 months ago. A quick look at a freshly sync node tells me that Bonsai Snap Sync are sitting around 1.1 TB now. @ahamlat have taken any measurements recently? I'd rather have 1.1 TB than 994GB (current change in this PR)

**Review by siladu — COMMENTED** [review]
Think it would be weird to update Snap without updating Fast, since Snap should be the same or superior I think. Wouldn't want people to think switching to Fast would be an improvement

**Comment by joaniefromtheblock** [comment]
Updates based on **[this doc](https://docs.google.com/document/d/1ao3ud8eUuFbWaELJIK_vpj8pSFPq62jIKHL_-_SOYo8/edit#heading=h.nvxmg96j3chn)** created by @ahamlat

**Comment by gfukushima** [comment]
I can't find the Forest snap numbers in that doc, @ahamlat did you spin up a forest snap as well?

**Comment by non-fungible-nelson** [comment]
LGFTM pending checkpoint

File: docs/public-networks/concepts/data-storage-formats.md [hunk 1]
```diff
@@ -81,10 +81,12 @@ The following table shows the ways you can [sync a full node](../get-started/con
 
 | Data storage format | Sync mode | Storage estimate | Can other nodes sync to your node? |
 | --- | --- | --- | --- |
-| Bonsai | Fast | 650 GB | No |
-| Bonsai | Snap | 650 GB | To be implemented |
-| Forest | Fast | 750 GB | Yes |
-| Forest | Snap | 750 GB | No |
+| Bonsai | Fast | 1140 GB | No |
+| Bonsai | Snap | 1090 GB | To be implemented |
+| Bonsai | Checkpoint | 833 GB | No |
```
Comment (siladu) [inline docs/public-networks/concepts/data-storage-formats.md:86]:
```suggestion
| Bonsai | Checkpoint | 840 GB | No |
```

I would round this up.

**Review by siladu — APPROVED** [review]
LGTM,. pending round up

File: docs/public-networks/concepts/data-storage-formats.md [hunk 2]
```diff
@@ -81,10 +81,12 @@ The following table shows the ways you can [sync a full node](../get-started/con
...
+| Forest | Fast | 1200 GB | Yes |
+| Forest | Snap | 1180 GB | No |
```
Comment (ahamlat) [inline docs/public-networks/concepts/data-storage-formats.md:88]: Where does this come from ?

**Review by ahamlat — APPROVED** [review]

**Review by alexandratran — APPROVED** [review]

**Pull request merged**